scrapers/dc/bills.py

```diff
@@ -26,546 +27,269 @@ class DCBillScraper(Scraper):
-            else:
-                self.logger.warning("Bad link for document {}".format(doc_name))
-                return
-
-            mimetype = "application/pdf" if doc_name.endswith("pdf") else None
-
-            # figure out if it's a version from type/name
-            possible_version_types = [
-                "SignedAct",
-                "Introduction",
-                "Enrollment",
-                "Engrossment",
-            ]
-            for vt in possible_version_types:
-                if vt.lower() in doc_name.lower():
-                    is_version = True
-                    doc_type = vt
-
-            if "amendment" in doc_name.lower():
-                doc_type = "Amendment"
-
-            if is_version:
-                bill.add_version_link(
-                    doc_type, doc_url, media_type=mimetype, on_duplicate="ignore"
-                )
-                continue
+                        # Votes
+                        if act["voteDetails"]:
+                            result = act["voteDetails"]["voteResult"]
+                            if result:
+                                status = self._vote_statuses[result.lower()]
+                                id_text = (
+                                    str(leg["legislationNumber"])
+                                    + "-"
+                                    + action_name
+                                    + "-"
+                                    + result
+                                )
+                                if id_text not in vote_ids:
+                                    vote_ids.append(id_text)
+                                    action_class = self.classify_action(action_name)
+                                    v = VoteEvent(
+                                        identifier=id_text,
+                                        chamber=actor,
+                                        start_date=action_date,
+                                        motion_text=action_name,
+                                        result=status,
+                                        classification=action_class,
+                                        bill=bill,
+                                    )
+                                    v.add_source(leg_listing_url)
+
+                                    yes_count = no_count = other_count = 0
+                                    for leg_vote in act["voteDetails"]["votes"]:
+                                        mem_name = leg_vote["councilMember"]
+                                        if leg_vote["vote"] == "Yes":
+                                            yes_count += 1
+                                            v.yes(mem_name)
+                                        elif leg_vote["vote"] == "No":
+                                            no_count += 1
+                                            v.no(mem_name)
+                                        else:
```

Comment:
what kinds of other votes were you seeing?  we might want to categorize them now

Reply:
Added in absent, recused (abstain), and Present (other) as vote categories.